Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -53,6 +53,7 @@
     private TagEditText mTagEditText;    /* Chips view editor for tags */
 
     private int         mBackgroundColor;
+    private boolean     mNoteChanged;
```

Comment:
So we need to separate `isContentChanged()` and `isNoteChanged()` methods out of `doSave()` method. Then the alert dialog will be shown if note is changed. By separating this methods we won't repeat the code to check note changes.